lib/action_policy/policy/reasons.rb

```diff
@@ -16,7 +16,7 @@ def add(policy_or_class, rule, details = nil)
         policy_class = policy_or_class.is_a?(Module) ? policy_or_class : policy_or_class.class
         reasons[policy_class] ||= []
 
-        if details.nil?
+        if details.nil? || details.empty?
```

Comment:
I could have use `details.blank?` but I'm not sure how it should depend on ActiveSupport or not.